src/properties/property-accessors.html

```diff
@@ -146,17 +148,40 @@
        * @protected
        */
       _setPendingProperty(property, value) {
-        let old = this.__data[property];
-        if (this._shouldPropChange(property, value, old)) {
+        let isPath = Polymer.Path.isPath(property);
+        let isObject = typeof value == 'object';
```

Comment:
This change adds a `Polymer.Path` dependency to `property-accessors` (up till now accessors was agnostic to path keys).  Refactor and pull up to `property-effects`?